plugins/woocommerce/client/legacy/js/admin/order-source-attribution-admin.js

```diff
@@ -0,0 +1,35 @@
+/*global woocommerce_admin_meta_boxes, wcTracks */
+jQuery( document ).ready( ( $ ) => {
+	'use strict';
+
+	// Stand-in wcTracks.recordEvent in case Tracks is not available (for any reason).
+	window.wcTracks = window.wcTracks || {};
+	window.wcTracks.recordEvent = window.wcTracks.recordEvent || ( () => {} );
+
+	// Handle the "Details" container toggle.
+	$( '.woocommerce-order-source-attribution-details-toggle' )
+		.on( 'click', ( e ) => {
+			const $this = $( e.target ).hasClass( 'woocommerce-order-source-attribution-details-toggle' ) ? $( e.target ) : $( e.target ).closest( '.woocommerce-order-source-attribution-details-toggle');
+			const $container = $this.closest( '.order-source-attribution-metabox' )
+											.find( '.woocommerce-order-source-attribution-details-container' );
+			var toggle = '';
```

Comment:
:nail_care: 
- We can cache `$( e.target )` not to do it twice, or actually not use it as...
- `.closest` takes the element itself as well if it matches
- we can use `let` instead of `var`

```suggestion
			const $this = $( e.target ).closest( '.woocommerce-order-source-attribution-details-toggle');
			const $container = $this.closest( '.order-source-attribution-metabox' )
											.find( '.woocommerce-order-source-attribution-details-container' );
			let toggle = '';
```